src/data/full-quiz.js

```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "What is the data type of (1) and (1,) ?",
+    Answer: "<class 'int'>  <class 'tuple'>",
+    Distractor1: "<class 'tuple'> <class 'int'>",

```

Comment:
I think we should change this to include a more obvious wrong answer like a data type of string. 
Proposed fix: 
 Distractor1: "<class 'str'> <class 'int'>",